tests/test_utils.py

```diff
@@ -2,7 +2,9 @@
 
 from networkx import Graph
 
-from agh_graphs.utils import sort_segments_by_angle, angle_with_x_axis
+from agh_graphs.utils import gen_name, sort_segments_by_angle, angle_with_x_axis
+
+visualize_tests = 'VISUALIZE_TESTS' in os.environ and os.environ['VISUALIZE_TESTS'] == 'true'
```

Comment:
Chyba przez pomyłkę wrzuciłeś tutaj zmiany z `tests/utils.py`.

Reply:
Nie przez przypadek, domergowąłem mastera do tego brancha

Reply:
No tak, tylko że w masterze nie ma tego kodu tutaj. Gdyby był to nie wyświetlałby się jako dodany. Teraz on jest w dwóch miejscach: `test_utils.py` i `utils.py`.